config/config.go

```diff
@@ -35,6 +35,8 @@ type HostCookie struct {
 	OptOutURL    string `mapstructure:"opt_out_url"`
 	OptInURL     string `mapstructure:"opt_in_url"`
 	OptOutCookie Cookie `mapstructure:"optout_cookie"`
+	// Cookie timeout in days
+	TTL int64 `mapstructure:"ttl"`
```

Comment:
Can we call this `ttl_days`? Less chance of screwing it up while editing the config.

Reply:
Done